docs/how_to_guides/msm_estimation_exercise.ipynb

```diff
@@ -0,0 +1,1419 @@
+{
```

Comment:
<ul><li>Please rename <code>num_boot</code>  and <code>sample_boot </code> to <code>n_bootstrap_samples</code>  and <code>n_observations_per_sample</code>  or something similar but more explicit.</li><li>Instead of <code>np.random.choice</code>,  can you use <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sample.html" target="_blank">https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sample.html</a>. After this <code>data.loc[ids_boot, :]</code>  is not necessary anymore, but in any case <code>data.loc[ids_boot]</code>  is sufficient. No need for the colon.</li><li><code>[]</code>  isntead of <code>list() </code> </li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Is it possible to sample only from one index level though? I couldn't figure out how.

Reply:
Good point :). Unfortunately, there is no `groupby("index_1").sample()` method.

Reply:
Okay thank you! Too bad, that would have definitely made the code there much nicer.